plugins/pulp_rpm/plugins/distributors/yum/publish.py

```diff
@@ -265,10 +265,12 @@ def __init__(self, transfer_repo, publish_conduit, config, distributor_type, **k
 
         last_published = publish_conduit.last_publish()
         last_deleted = repo.last_unit_removed
+        force_full = config.get('force_full', False)
         date_filter = None
 
         if last_published and \
-                ((last_deleted and last_published > last_deleted) or not last_deleted):
+                ((last_deleted and last_published > last_deleted) or not last_deleted) and \
```

Comment:
Similar simplification of this if statement as suggested here: https://github.com/pulp/pulp/pull/2548#discussion_r60284025